src/components/tooltip/tooltip.svelte

```diff
@@ -31,7 +31,8 @@
   export let fadeDuration: number = 0
 
   /** Called when the visibility of the tooltip is changed */
-  export let onVisibilityChange: (detail: { visible: boolean }) => void = undefined
+  export let onVisibilityChange: (detail: { visible: boolean }) => void =
```

Comment:
Sometimes `prettier` is obnoxious.

Reply:
You can't tell prettier it aint prettier :D